src/Operation/Watch.php

```diff
@@ -79,22 +92,44 @@ class Watch implements Executable
      *  * resumeAfter (document): Specifies the logical starting point for the
      *    new change stream.
      *
+     *    Using this option in conjunction with "startAtOperationTime" will
+     *    result in a server error. The options are mutually exclusive.
```

Comment:
Per [`startatoperationtime`](https://github.com/mongodb/specifications/blob/master/source/change-streams/change-streams.rst#startatoperationtime) in the spec, we're not supposed to raise a client-side error if both options are used.